src/plugins/intel_cpu/tests/functional/utils/cpu_test_utils.cpp

```diff
@@ -542,4 +536,22 @@ CPUTestsBase::deduce_expected_precision(const ov::element::Type& opPrecision,
     return deducedType;
 #endif
 }
+
+void CheckNodePrecisionsWithType(const ov::CompiledModel& compiledModel,
+                                 const std::string& nodeType,
+                                 ov::element::TypeVector&& inPrecisions,
+                                 ov::element::TypeVector&& outPrecisions) {
```

Comment:
NIT. When the refs intended to be used read only, it's better to use a constant lvalue reference.

Reply:
You are right, corrected, thanks